regression/symtab2gb/Makefile

```diff
@@ -0,0 +1,27 @@
+default: tests.log
+
+include ../../src/config.inc
+include ../../src/common
+
+test:
+	@../test.pl -e -p -c '../chain.sh $(exe) ../../../src/symtab2gb/symtab2gb ../../../src/cbmc/cbmc'
+
+tests.log:
+	@../test.pl -e -p -c '../chain.sh $(exe) ../../../src/symtab2gb/symtab2gb ../../../src/cbmc/cbmc'
+
+show:
+	@for dir in *; do \
+		if [ -d "$$dir" ]; then \
+			vim -o "$$dir/*.c" "$$dir/*.out"; \
```

Comment:
$VISUAL or $EDITOR, rather than "vim" ?

Reply:
The `-o` option is specific to `vim`, and it's the same bit of code we have everywhere else. But I don't know whether anyone ever users this target?!

Reply:
Ah, this was the first time I've actually noticed this target - never used it myself!

Reply:
Ditto on never using it. It's honestly a bit of cargo culting, I was kind of assuming _someone_ around here liked to work that way. But I certainly don't mind removing it, I have no attachment to it.

Reply:
I'd vote for having a separate PR that removes all of those. If someone is still using them, we can then have that discussion on the PR.